src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerAssemblyLoader.cs

```diff
@@ -164,82 +170,95 @@ public Assembly LoadFromPath(string originalAnalyzerPath)
 
             lock (_guard)
             {
-                _analyzerAssemblyInfoMap[originalAnalyzerPath] = (assemblyName, realPath, resourceAssemblyCultureNames);
+                _analyzerAssemblyInfoMap[originalAnalyzerPath] = (assemblyName, realPath);
             }
 
-            return (assemblyName, realPath, resourceAssemblyCultureNames);
+            return (assemblyName, realPath);
+        }
+
+        internal string? GetSatelliteInfoForPath(string originalAnalyzerPath, CultureInfo cultureInfo)
```

Comment:
Consider using more descriptive name. It is unclear what the return value is, the method's name only suggests it's some "info". Alternatively, add a doc comment.

```suggestion
        internal string? GetSatelliteAssemblyPath(string originalAnalyzerPath, CultureInfo cultureInfo)
```